src/chrome/content/rules/ebay.nl.xml

```diff
@@ -0,0 +1,205 @@
+	<target host="epnt.ebay.nl" />
+	<target host="epsvc.ebay.nl" />
+	<target host="fedauth.ebay.nl" />
+	<target host="fedsignin.ebay.nl" />
+	<target host="feedback.ebay.nl" />
+	<target host="frame.ebay.nl" />
+	<target host="fundinginstrument.ebay.nl" />
+	<target host="fyp.ebay.nl" />
+	<target host="fyp2.ebay.nl" />
+	<target host="gha.ebay.nl" />
+	<target host="invoice.ebay.nl" />
+	<target host="item.ebay.nl" />
+	<target host="www.kgvrfn.ebay.nl" />
+	<target host="m.ebay.nl" />
+	<target host="www.m.ebay.nl" />
+	<target host="reg.m.ebay.nl" />
+	<target host="scgi.m.ebay.nl" />
+	<target host="signin.m.ebay.nl" />
+	<target host="marketing.ebay.nl" />
+	<target host="mbuy.ebay.nl" />
+	<target host="mesg.ebay.nl" />
+	<target host="notifyweb.ebay.nl" />
+	<target host="ocs.ebay.nl" />
+	<target host="ocsnext.ebay.nl" />
+	<target host="ocsrest.ebay.nl" />
+	<target host="ocswf.ebay.nl" />
+	<target host="offer.ebay.nl" />
+	<target host="ofr.ebay.nl" />
+	<target host="pages.ebay.nl" />
+	<target host="paisapay.ebay.nl" />
+	<target host="pay.ebay.nl" />
+	<target host="payments.ebay.nl" />
+	<target host="apps.payments.ebay.nl" />
+	<target host="checkout.payments.ebay.nl" />
+	<target host="guestcheckout.payments.ebay.nl" />
+	<target host="paymentscmd.ebay.nl" />
+	<target host="www.picupload.ebay.nl" />
+	<target host="nw.pp.ebay.nl" />
+	<target host="www.nw.pp.ebay.nl" />
+	<target host="ppm1.ebay.nl" />
+	<target host="ppm3.ebay.nl" />
+	<target host="pulsar.ebay.nl" />
+	<target host="qu.ebay.nl" />
+	<target host="rebulk.ebay.nl" />
+	<target host="reg.ebay.nl" />
+	<target host="res.ebay.nl" />
+	<target host="resolutioncenter.ebay.nl" />
+	<target host="reward.ebay.nl" />
+	<target host="rover.ebay.nl" />
+	<target host="www.rsvp.ebay.nl" />
+	<target host="www.bizpolicy.sandbox.ebay.nl" />
+	<target host="checkout.sandbox.ebay.nl" />
+	<target host="community.sandbox.ebay.nl" />
+	<target host="fedauth.sandbox.ebay.nl" />
+	<target host="mesg.sandbox.ebay.nl" />
+	<target host="scgi.sandbox.ebay.nl" />
+	<target host="signin.sandbox.ebay.nl" />
+	<target host="scgi.ebay.nl" />
+	<target host="securefeedback.ebay.nl" />
+	<target host="secureportal.ebay.nl" />
+	<target host="securepromo.ebay.nl" />
+	<target host="secureregistration.ebay.nl" />
+	<target host="sellerstandards.ebay.nl" />
+	<target host="sellerupdate.ebay.nl" />
+	<target host="sellvrfn.ebay.nl" />
+	<target host="www.sellvrfn.ebay.nl" />
+	<target host="settings.ebay.nl" />
+	<target host="shiptrack.ebay.nl" />
+	<target host="signin.ebay.nl" />
+	<target host="signinalert.ebay.nl" />
+	<target host="sofe.ebay.nl" />
+	<target host="spd.ebay.nl" />
+	<target host="www.sps.ebay.nl" />
+	<target host="srtm.ebay.nl" />
+	<target host="viv.ebay.nl" />
+	<target host="wantitnow.ebay.nl" />
+
+	<!-- fix http redirect -->
+	<rule from="^http://ebay\.nl/"
+		to="https://www.ebay.nl/" />
+
+	<rule from="^http:"
+		to="https:" />
+</ruleset>
```

Comment:
I miss some hosts here:
```
ocsnext.ebay.nl
pages.ebay.nl

Reply:
what do you mean? they're non-functional?

Reply:
Sorry, I just found them in your commit.